libraries/AC_WPNav/AC_Circle.h

```diff
@@ -34,10 +35,17 @@ class AC_Circle
 
     /// get_radius - returns radius of circle in cm
     float get_radius() { return _radius; }
+
     /// set_radius - sets circle radius in cm
-    void set_radius(float radius_cm) { _radius = radius_cm; }
+    void set_radius(float radius_cm);
+
+    /// get_rate - returns rate in deg/sec
+    float get_rate() { return _rate; }
```

Comment:
we can make get_rate() and get_rate_current() const functions.  I.e. "float get_rate() const { return _rate; }

nitpick: there's a small spacing issue with get_rate_current().  there should be extra spaces between the curly brackets and the "r" and ";"

Reply:
Done